openmc/macroscopic.py

```diff
@@ -0,0 +1,85 @@
+from numbers import Integral
+import sys
+
+from openmc.checkvalue import check_type
+
+if sys.version_info[0] >= 3:
+    basestring = str
+
+
+class Macroscopic(object):
```

Comment:
I'm confused - what does this class offer beyond `Nuclide` and when would one want to use it?